build

```diff
@@ -29,6 +31,15 @@ while :; do
         -c|--use-cache)
             CACHE_FLAG=''
             ;;
+        -b|--base-image-tag)
+            if [[ -z $2 ]]; then
+                usage
+                printf 'ERROR: No TAG specified after the %s flag.\n' "$1" >&2
+                exit
+            fi
+            BUILD_ARGS="--build-arg BASE_TAG=$2"
```

Comment:
You could make this an array of args and use `"${BUILD_ARGS[@]}"` below?

Reply:
Done